sympy/vector/functions.py

```diff
@@ -173,6 +174,36 @@ def directional_derivative(field, direction_vector):
         return S(0)
 
 
+def laplacian(expr):
+    """
+    Returns the laplacian of the given field computed wrt
```

Comment:
*wrt* means `with respect to`, right?
For base scalars it sounds strange to me.
I suggest:
*Returns the laplacian of the given field computed **in terms of** the base scalars of the given coordinate system*

Reply:
Yes, 'wrt' means 'with respect to' here. I think you are right, *in terms of* is better suited here.

How about we instead have *Return the sum of second order partial derivatives of 'f' with respect to the base scalars* ?

Reply:
It's good proposition but I'd prefer something which use concept from vector analysis.